ci/buildkite.yml

```diff
@@ -20,7 +20,7 @@ steps:
     timeout_in_minutes: 60
   - command: ". ci/rust-version.sh; ci/docker-run.sh $$rust_stable_docker_image ci/test-stable.sh"
     name: "stable"
-    timeout_in_minutes: 40
+    timeout_in_minutes: 50
```

Comment:
Is this needed?

Reply:
Yeah the release builds take a lot more time now :P